
class ValidationError(Exception):
    pass


class CoercionError(Exception):
    pass
